docs/ONCALL.adoc

```diff
@@ -0,0 +1,13 @@
+== Dates Oncall:
+
+=== Summary/Notes:
+
+* Use this section to put high level context that may be helpful to the next person on-call
```

Comment:
It would be great to provide an illustrative example for this. For example:

```
Last week (Friday night) there was one product launch and two large releases for existing products/services (Tuesday night). Many of the incidents triggered were around these code changes in production. Some (~30%) were false positives due to outmoded status scripts for infrastructure.
```